source/releasenotes/2024-01-25-pantheon-release-notes-a-new-year-a-new-chapter.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Pantheon release notes - A new year, a new chapter
+published_date: "2024-01-25"
+categories: [documentation, user-interface]
+---
+
+As we embark on a new year, we're excited to unveil a transformation. This January marks the introduction of **Pantheon release notes**, a rebranding of our changelog. Our aim is to elevate your platform experience by providing even more valuable insights into product updates.
+
+**What's new?**
```

Comment:
@IngridKwok what do you think about using H3s instead of bolded text for each subsection of a given entry?  
<img width="976" alt="Screenshot 2024-01-25 at 1 24 09 PM" src="https://github.com/pantheon-systems/documentation/assets/10119525/7885ccba-94da-4933-85fc-9a272a7127e6">